apps/web/cypress/tests/notification-editor/steps-actions.spec.ts

```diff
@@ -5,11 +5,22 @@ describe('Workflow Editor - Steps Actions', function () {
     cy.initializeSession().as('session');
   });
 
+  const interceptEditTemplate = () => {
+    cy.intercept('**/notification-templates/**').as('getTemplate');
```

Comment:
The alias doesn't seem to match the function name (the goal of the function).

Reply:
yeah I'll rename it, thanks!